vars/DevelopPipeline.groovy

```diff
@@ -38,8 +38,8 @@ def call(name, idl_name, module_name, pre_commit_flags="", required_idl=[], buil
         }
         environment {
             user_ci = credentials('lsst-io')
-            LTD_USERNAME="${user_ci_USR}"
-            LTD_PASSWORD="${user_ci_PSW}"
+            LTD_USERNAME='{$user_ci_USR}'
+            LTD_PASSWORD='{$user_ci_PSW}'
```

Comment:
Are you using these environment variables? I see $user_ci_USR/PSW in the code above. If you are not using these, but want to, I suggest changing the code above. If you don't need or want them, then I suggest removing these two lines.

Reply:
My reasoning to keep them was because the docs state that these provide a default username and password for -u and -p and I figured that it would be a good backup. There shouldn't be any harm in keeping them here.

Reply:
The harm is in making the code confusing by having two different ways to do the same thing.

If this default works then I suggest not using `$user_ci` elsewhere in this file.